privaterelay/middleware.py

```diff
@@ -8,11 +10,42 @@
 from django.shortcuts import redirect
 
 import markus
+from csp.middleware import CSPMiddleware
 from whitenoise.middleware import WhiteNoiseMiddleware
 
 metrics = markus.get_metrics("fx-private-relay")
 
 
+CSP_NONCE_COOKIE_URLS = ["/", "/premium", "/faq", "/accounts/profile/"]
```

Comment:
Add "/phone/", and add the trailing slash for `/premium/` and `/faq/`

Reply:
7f0fb8b3280f86fc5b9e94acbf77792839ca04d6